ide-support/lsp/lsp-server/src/main/java/io/helidon/lsp/server/management/MavenSupport.java

```diff
@@ -123,7 +129,11 @@ public Set<Dependency> dependencies(final String pomPath, int timeout) {
                     result = GSON.fromJson(in, new TypeToken<Set<Dependency>>() {
                     }.getType());
                 } catch (IOException e) {
-                    LOGGER.log(Level.SEVERE, "Error when executing the maven command - " + lspMvnDependenciesCommand, e);
+                    String message = "Error when executing the maven command - "
```

Comment:
The normal thing to do here is to propagate the exception as a `UncheckedIOException`, it will be propagated as the cause of an ExecutionException with the `.get()` call below.